docs/misc/config.md

```diff
@@ -15,16 +15,27 @@ A configuration can be created using a subtype of `IConfigSpec`. NeoForge implem
 `ModConfigSpec.Builder#configure` is typically used with a `static` block and a class that takes in `ModConfigSpec.Builder` as part of its constructor to attach and hold the values:
 
 ```java
-// In some config class
-ExampleConfig(ModConfigSpec.Builder builder) {
-    // Define values here in final fields
-}
+//Store the config properties as finals
+final ModConfigSpec.ConfigValue<String> testValue;
```

Comment:
I would move this down to the example section too. Also why is this final and not public? I doubt only this package would make sense to access the config values. I believe there's no issue with this being public now since the values are cached and invalidated as needed.

Reply:
Moved and access changed